lib/Quantization/Quantization.cpp

```diff
@@ -99,6 +99,10 @@ static bool canBeQuantized(const Node *node) {
     auto *gather = cast<GatherNode>(node);
     return gather->getData().getElementType() == ElemKind::FloatTy;
   }
+  case Kinded::Kind::SoftMaxNodeKind: {
```

Comment:
as long as we quantize node, we should ensure proper support at least by interpreter backend.